examples/resources/plugins/teleport-msteams.toml

```diff
@@ -0,0 +1,49 @@
+# Example ms teams plugin configuration TOML file
+
+# If true, recipients existence got checked on plugin start
+# When a recipient is checked, the app is installed for the user if it was not already.
+# This takes some time and does not fits well with HTTP timeouts, hence preload is necessary when a new
+# recipient is added.
+preload = true
```

Comment:
I think the comment here needs a bit more context. Maybe move the comment text into a `Details` box where we explain different configuration fields. In particular, I'm wondering:
- "recipient": what does this mean in this context?
- what does "does not fits well with HTTP timeouts" mean? The plugin might fail to start due to recipient checking exceeding HTTP timeouts on the MS Teams API backend?